CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(nps_uw_sensors)
```

Comment:
Project name should be same as the repository - `uw_sensors_gazebo`

Reply:
Perhaps we should rename the repository to nps_uw_sensors_gazebo until it gets adopted.  Per https://www.ros.org/reps/rep-0144.html: if a package is specialized by an entity (lab, company, ...), prepend the name of the entity. Once the package is commonly used, owned and maintained, that name can be dropped.

We may also want to open up the scope of this repository and rename it to uw_sensors, then create subdirectory nps_uw_sensors_gazebo and move CMakeLists.txt and all these subdirectories down under that.

Reply:
yes - let's rename to make consistent with the REP.

Reply:
Okay, I'll rename this repository to nps_uw_sensors_gazebo.  Would it be useful to additionally open up the scope of this repository and rename it to nps_uw_sensors?  If so, we would create subdirectory nps_uw_sensors_gazebo and move CMakeLists.txt and all these subdirectories down under that.

Reply:
We'll keep scope as is.